vernac/vernacextend.ml

```diff
@@ -91,7 +92,7 @@ let warn_deprecated_command =
 
 (* Interpretation of a vernac command *)
 
-let call opn converted_args ~atts =
+let type_vernac opn converted_args ~atts =
```

Comment:
This name is bad, this doing no typing at all, but it is an interpretation function.